test/tests.bats

```diff
@@ -956,7 +956,7 @@ EOF
   assert_success
 
   value=$(grep setup_email_add@example.com "$(private_config_path mail)/postfix-accounts.cf" | awk -F '|' '{print $1}')
-  [ "${value}" = "setup_email_add@example.com" ]
+  [[ ${value} = "setup_email_add@example.com" ]]
```

Comment:
I'm not sure if this matters or not, I'm just used to seeing equality checks with `==` instead of `=`, is there any difference here with bash? Otherwise probably more appropriate to use `==`?

```suggestion
  [[ ${value} == "setup_email_add@example.com" ]]
```

Reply:
I don't even know what a single `=` does, so I guess, **yes**, we should definitely go for it.

Reply:
Single and double `=` behave the same, when it comes to string comparison.
However, `=` is defined by POSIX and more portable, while `==` is a bash specific and does not work e.g. with `/bin/sh`.